cookbook/movie_bot/recommend_movies.py

```diff
@@ -0,0 +1,21 @@
+"""A script for recommending movies."""
+import os
+
+from prompts import MovieRecommendationPrompt
+
+from mirascope import OpenAIChat
+
+os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"
+
+
+def recommend_movies(genre: str):
+    """Run the movie recommendation script."""
+    model = OpenAIChat()
```

Comment:
do you need to add api_key here?

Reply:
not if you set `os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"` which I put at the top